src/RequestHandler.h

```diff
@@ -74,5 +74,10 @@ class RequestHandler {
       const std::shared_ptr<Subscriber<Payload>>& subscriber) noexcept = 0;
   virtual void onSubscriberResumed(
       const std::shared_ptr<Subscriber<Payload>>& subscriber) noexcept = 0;
+
+  // TODO (T17774014): Move to separate interface
```

Comment:
Is there anything depending on this diff that makes us having to delay this? It seems like splitting these is a <1h job.